recipes/hachoir-subfile/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{%set name = "hachoir-subfile" %}
+{%set version = "0.5.3" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "b8e4933e566524804382c8f8a9a9b58a4e25ad24ebf75beb0da5d8705c61f0d6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - hachoir-core
+    - hachoir-parser
+    - hachoir-regex
+
+test:
+  imports:
+    - hachoir_subfile
+    - hachoir_subfile.data_rate
+    - hachoir_subfile.output
+    - hachoir_subfile.pattern
+    - hachoir_subfile.search
+    - hachoir_subfile.version
+
+about:
+  home: http://hachoir.org/wiki/hachoir-subfile
```

Comment:
This link is broken--and hachoir.org doesn't appear to have any relation to the software project.